boot.php

```
@@ -18,37 +33,41 @@
-rex_minibar::getInstance()->addElement(new rex_minibar_element_syslog());
+/** TODO: Instanz nur einmal abrufenn und als Variable nutzen "$minibar = Minibar::getInstance(); $minibar->addelemene..." */
+Minibar::getInstance()->addElement(new System());
+Minibar::getInstance()->addElement(new Time());
+Minibar::getInstance()->addElement(new Syslog());
 rex_api_function::register('mbclrcache', ClearCache::class);
+rex_api_function::register('minibar', LazyLoader::class);
 
 // URL2/YForm Element für alle Backend-Bereiche verfügbar machen
 if (rex::isFrontend()) {
-rex_minibar::getInstance()->addElement(new rex_minibar_element_url2_yform());
+Minibar::getInstance()->addElement(new Url2Yform());
```

PHPStan (2.1.51)

[error] Call to an undefined static method FriendsOfRedaxo\Minibar\Minibar::getInstance().


(staticMethod.notFound)